editor/utils/dom.js

```diff
@@ -150,7 +150,7 @@ export function placeCaretAtHorizontalEdge( container, isReverse ) {
 
 	if ( includes( [ 'INPUT', 'TEXTAREA' ], container.tagName ) ) {
 		container.focus();
-		if ( isReverse ) {
+		if ( ! isReverse ) {
```

Comment:
Minor: When negating, I usually prefer to swap the blocks for readability. E.g.

```js
! isSomething ? a() : b()
```

to 

```js
isSomething ? b() : a()
```

Reply:
Ok I will make that minor change and then merge.